markdown/articles/2021/08/Non_blue_fair_decks.txt

```diff
@@ -0,0 +1,115 @@
+---
+authors: OtterLady
+banner: /assets/stinging-study.c21.jpg
+title: “Matchup Guides: Non Blue Fair Decks”
+---
+
+Gude!
+
+This time i want to talk how our Gameplan against Non Blue Fair Decks looks like and how to execute it.
+All these matchups have one common point, a fast Doomsday is always good and the post Doomsday part of the game is far more interesting.
+
+!EXCLAIMER! I don't give any sideboard guides in these articles. Sideboard guides don't help anyone learning to play a deck, because you first have to know how to play the deck to know how to sideboard correctly.
+
+##DnT
+
+###Gameplan
+Slam early and build your pile around mana and card draw denial.
+
+###Cards to keep in mind
+####Maindeck
+<row variant="hand"> {{!Kaldra Compleat}} {{!Spirit of the Labyrinth}} {{!Rishadan Port}} {{!Wasteland}} {{!Solitude}}</row>
+
+####Sideboard
+<row variant="hand"> {{!Deafening Silence}} {{!Mindbreak Trap}} {{!Surgical Extraction}</row>
+
+###Gameplay
+####Pre Doomsday
+Against DnT resolving an early {{Doomsday}} is the safest way to win. Mulliganing for a Turn 1 or Turn 2 Doomsday is a good starting point.
```

Comment:
Add blank lines after paragraphs. Otherwise, they won't be considered *paragraphs* but part of the same block of text (no new line).

If you actually want a simple carriage return (new line but no spacing), append 2 consecutive spaces at the end of the line in your file.

```suggestion
Against DnT resolving an early {{Doomsday}} is the safest way to win. Mulliganing for a Turn 1 or Turn 2 Doomsday is a good starting point.

```

Reply:
added space were necessary